ftl/common/args.py

```diff
@@ -123,24 +123,51 @@ def base_parser():
 
 node_flgs = []
 php_flgs = []
-python_flgs = ['python_version']
+python_flgs = ['python_cmd', 'pip_cmd', 'venv_cmd']
 
 
 def extra_args(parser, opt_list):
     opt_dict = {
-        'python_version': [
-            '--python-version', {
+        'python_cmd': [
+            '--python-cmd', {
                 "dest":
-                'python_version',
+                'python_cmd',
                 "action":
                 'store',
                 "default":
                 "python2.7",
                 "help":
-                'The python version to use, formatted as pythonX.X \
+                'The python command to be run \
                 (ex: python2.7)'
             }
         ],
+        'pip_cmd': [
```

Comment:
nit: I find these easier to read if you collapse the key: value to one line.

Reply:
done